packages/plugin-snowplow-react-native/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@itly/plugin-snowplow-react-native",
+  "version": "2.0.5",
+  "description": "Snowplow React Native Plugin for Iteratively SDK",
+  "main": "dist/cjs/index.js",
+  "types": "dist/cjs/index.d.ts",
+  "module": "dist/esm/index.js",
+  "scripts": {
+    "prepublish": "yarn run build",
+    "build": "tsc -p tsconfig.json && tsc -p tsconfig-esm.json",
+    "test": "jest --runInBand"
+  },
+  "files": [
+    "CHANGELOG.md",
+    "README.md",
+    "dist/",
+    "package.json"
+  ],
+  "keywords": [
+    "itly",
+    "sdk",
+    "iteratively",
+    "plugin"
+  ],
+  "author": "Iteratively",
+  "license": "ISC",
+  "devDependencies": {
+    "@itly/sdk": "2.0.5",
+    "@snowplow/react-native-tracker": "^0.1.3",
+    "typescript": "^3.9.3"
+  },
+  "peerDependencies": {
+    "@itly/sdk": "^2.0.0",
+    "@snowplow/react-native-tracker": "^0.1.3"
```

Comment:
Can you please add install instructions to the README. Describing to `yarn add @snowplow/react-native-tracker` to the user project, react-native link, and any thing else needed for setup.